### PR TITLE
Initial tests

### LINKED ISSUE #19: write some tests for models and training
**What's your idea?**
We should improve our test coverage in this repo.

**Describe the solution you'd like**
Simple tests for:
- model can accept tensors of expected shape
- model can decrease the loss through a single optimization step
- feature processing code is tested

### PR DESCRIPTION
**Pull request by jeremyjordan**
## What does this PR do?

Fixes #19

### CODE REVIEW

File: tests/models/test_baseline.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+@pytest.mark.parametrize("batch_size, img_height, img_width", [(1, 256, 256), (8, 100, 300), (4, 32, 32)])
+def test_expected_input_shape(network: torch.nn.Module, batch_size: int, img_height: int, img_width: int):
+    example_input_array = torch.zeros(batch_size, 3, img_height, img_width)
+    _ = network(example_input_array)
+
+
+@pytest.mark.parametrize(
+    "batch_size, img_height, img_width",
+    [
+        (1, 4, 4),  # TODO: find exact boundary
```
Comment (huffmanjohnf) [inline tests/models/test_baseline.py]: Should we identify this before merging?

**Review by huffmanjohnf — APPROVED** [review]

**Review by huffmanjohnf — APPROVED** [review]
lgtm
